x-pack/platform/plugins/shared/stack_connectors/common/openai/schema.ts

```diff
@@ -34,6 +36,32 @@ export const ConfigSchema = schema.oneOf([
     defaultModel: schema.string(),
     headers: schema.maybe(schema.recordOf(schema.string(), schema.string())),
   }),
+  // Added PKI OpenAI schema
+  schema.object({
+    apiProvider: schema.oneOf([schema.literal(OpenAiProviderType.PkiOpenAi)]),
+    apiUrl: schema.string(),
+    defaultModel: schema.string({ defaultValue: DEFAULT_OPENAI_MODEL }),
+    certPath: schema.string({
```

Comment:
I'm still a bit confused because AFAICT, this sort of config could only be used by a customer in an on-premise deployment - it can't be used in any cloud deployments because customers have no access to the file system.

As an example, for the action plugin config, we allow ca file names OR their data to be used: https://github.com/elastic/kibana/blob/ba49a0a83cd5aaaf5ed254eb4123e52048c2d6d9/x-pack/platform/plugins/shared/actions/server/config.ts#L47-L64

That will work in cloud ...